src/main/java/seedu/address/model/session/Interval.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.session;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Recurring Session's interval.
```

Comment:
LGTM. But was thinking if we can be more explicit that the interval is in terms of the number of days.

Reply:
yep!